Makefile

```diff
@@ -415,3 +422,53 @@ docs-sync:
 	SRC=app/vmctl/README.md DST=docs/vmctl.md ORDER=8 $(MAKE) copy-docs
 	SRC=app/vmgateway/README.md DST=docs/vmgateway.md ORDER=9 $(MAKE) copy-docs
 	SRC=app/vmbackupmanager/README.md DST=docs/vmbackupmanager.md ORDER=10 $(MAKE) copy-docs
+
+enterprise-remote-check:
+ifeq ($(ENTERPRISE_REMOTE_EXISTS), 0)
+	$(error enterprise remote is missing)
+endif
+
+create-tags: enterprise-remote-check
+	git fetch --all
+	git switch enterprise && git tag -s $(TAG)-enterprise -m "$(TAG)"
```

Comment:
It is better to use `git checkout` instead of `git switch`, since `git switch` is unavailable in old versions of `git`, which are included in popular Linux distributives. For example, my Ubuntu 18.04 contains git v2.17.1, while `git switch` is available only starting from git v2.23.

Reply:
okie